kernel/README.md

```diff
@@ -44,6 +44,7 @@ Commands:
 Options:
 
 	-c <path>   : Path to config file to build a the kernel.
+	-b          : Enable BPF related features in kernel.
```

Comment:
Nit: This hasn't been added to the list of options alphabetically - it should come first.